packages/core/src/components/dropdown/test/multiselect-filter/dropdown.e2e.ts

```diff
@@ -0,0 +1,28 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/dropdown/test/multiselect/index.html';
+
+test.describe.parallel('tds-dropdown-multiselect-filter', () => {
+  test('When focusing on the input it should clear itself', async ({ page }) => {
+    await page.goto(componentTestPath);
+
+    // Click the dropdown button
+    const dropdownButton = page.getByRole('button', { name: 'Placeholder' });
+    await dropdownButton.click();
+
+    const dropdownListElementOneButton = page
+      .getByText(/Option 1/)
+      .filter({ has: page.getByRole('checkbox') });
+
+    await expect(dropdownListElementOneButton).toHaveCount(1);
+
+    await dropdownListElementOneButton.click();
+    await expect(dropdownListElementOneButton).toHaveAttribute('aria-selected');
+
+    const inputText = page.getByRole('button', { name: /Option 1/ });
+    await expect(inputText).toHaveCount(1);
+
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
+  });
+});
```

Comment:
Should we add a test that checks that the input field has been cleared once focused but still displays the selected option as well? Maybe add another screenshot for that too? :)

Reply:
Also, we should start with adding a test for dark mode everytime!

Reply:
I agree with @nathalielindqvist . A test which assure that the input field has been cleared once focus sound good.

Reply:
@timrombergjakobsson, could you clarify why adding a test for dark mode is necessary? From my understanding, the dark mode logic is handled outside the component itself and doesn't directly impact the component's internal logic.

Reply:
@ckrook this was something that we started discussing that we should start adding screenshots for dark mode to verify that it looks correct. Although we have not really been doing it in PR's  lately, so maybe we should have a new discussion regarding if we should do it or not. What do you say @theJohnnyMe?